File: CS/MvcApp_Theme/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using DevExpress.Web.Mvc;

namespace MvcApp_Theme.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            ViewBag.Message = "Welcome to DevExpress Extensions for ASP.NET MVC!";
            ThemesModel themeCollection = new ThemesModel() { Name = "Aqua", Theme = CommonThemes.Aqua };

            return View(themeCollection);
        }

        [HttpPost]
        public ActionResult PostTheme([ModelBinder(typeof(MyEditorsBinder))] ThemesModel themeCollection) {
            ViewBag.Message = String.Format("Current theme: {0}", themeCollection.Theme);
            return View("Index", themeCollection);
        }

        public ActionResult CalendarPartial() {
            return PartialView("CalendarPartial");
        }
    }
}

File: VB/MvcApp_Theme/Controllers/HomeController.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Namespace MvcApp_Theme.Controllers
    Public Class HomeController
        Inherits Controller

        Public Function Index() As ActionResult
            ViewBag.Message = "Welcome to DevExpress Extensions for ASP.NET MVC!"
            Dim themeCollection As New ThemesModel() With { _
                .Name = "Aqua", _
                .Theme = CommonThemes.Aqua _
            }

            Return View(themeCollection)
        End Function

        <HttpPost> _
        Public Function PostTheme(<ModelBinder(GetType(MyEditorsBinder))> ByVal themeCollection As ThemesModel) As ActionResult
            ViewBag.Message = String.Format("Current theme: {0}", themeCollection.Theme)
            Return View("Index", themeCollection)
        End Function

        Public Function CalendarPartial() As ActionResult
            Return PartialView("CalendarPartial")
        End Function
    End Class
End Namespace
